spec/mondrian_spec.rb

```diff
@@ -540,4 +540,62 @@ def nonempty_crossjoin_from_cube(cube_name)
       result.profiling_timing_string.should_not include("[Measures].[Unit Sales 2]")
     end
   end
+
+  describe "LinRegR2" do
+    it "should be 0" do
```

Comment:
It is not clear from test names "should be 0", "should be 0.52", "should be 1.0" what are they really testing. A test should read like a specification, now "LinRegR2 should be 0" does not make any sence.

Please rewrite them and specify what they are really testing. From the test implementation it is also not quite understandable why we are expecting this result.

Also It seems that the test depends that `[Customers].[Country].Members` have exactly 3 members.